indico/modules/designer/placeholders.py

```diff
@@ -422,6 +422,7 @@ class RegistrationFormFieldPlaceholder(DesignerPlaceholder):
     def __init__(self, field):
         self.field = field
         self.name = f'field-{field.id}'
+        self.is_image = self.field.input_type == 'picture'
```

Comment:
Nice that this works. I was somehow thinking the signal checks it on the class itself and not on an instance of it. But with instances it's of course easy to handle...